ansibullbot/utils/extractors.py

```diff
@@ -493,77 +494,63 @@ def metadata(self):
             self._METADATA = self.get_module_metadata()
         return self._METADATA
 
-    def get_module_authors(self):
-        """Grep the authors out of the module docstrings"""
+    @property
+    def docs(self):
+        if self._DOCSTRING is None:
```

Comment:
Pls reverse this check to be in a "guard expression" style. This will allow dedenting huge chunk of code. Flat src is better readable.
```suggestion
        if self._DOCSTRING is not None:
            return self._DOCSTRING
```